### PR TITLE
chore(docs): Fix docs starting with n

### PR DESCRIPTION
**Pull request by brkn**
## Description

- Checked all docs starting with `n`.
- Fixed heading levels
- Changed `we/our` wordings to `you/your`
- Cleaned some passive voice uses

## Related Issues

Related to #18284

### CODE REVIEW

File: docs/docs/node-creation.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 title: Node Creation
 ---
 
-Nodes are created by calling the [createNode](/docs/actions/#createNode) action. Nodes can be any object.
+Calling the [createNode](/docs/actions/#createNode) action creates nodes. Which can be any object.
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]:
```suggestion
Calling the [createNode](/docs/actions/#createNode) action creates Node objects.
```

File: docs/docs/node-creation.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 
-A node is stored in redux under the `nodes` namespace, whose state is a map of the node ID to the actual node object.
+Redux stores nodes under the `nodes` namespace, whose state is a map of the node ID to the actual node object.
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]:
```suggestion
Redux stores nodes in the `nodes` namespace which is a map of ID to node Object.
```

File: docs/docs/node-creation.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
+The creation of nodes occurs primarily in the [sourceNodes](/docs/node-apis/#sourceNodes) bootstrap phase. Nodes created during this phase are top level nodes. I.e, they have no parent. source plugins setting the node's `parent` field to `null` represents this. Nodes created via transform plugins (who implement [onCreateNode](/docs/node-apis/#onCreateNode)) will have source nodes as their parents, or other transformed nodes. For a rough overview of what happens when source nodes run, see the [traceID illustration](/docs/how-plugins-apis-are-run/#using-traceid-to-await-downstream-api-calls).
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]:
```suggestion
The creation of nodes occurs primarily in the [sourceNodes](/docs/node-apis/#sourceNodes) bootstrap phase. Nodes created during this phase are top level nodes (i.e. they have no parent). Source plugins set the node's `parent` field to `null` to represent this. Nodes created via transformer plugins (which implement [onCreateNode](/docs/node-apis/#onCreateNode)) set source nodes as their parents, or other transformed nodes. For a rough overview of what happens when `sourceNodes` runs, see the [traceID illustration](/docs/how-plugins-apis-are-run/#using-traceid-to-await-downstream-api-calls).
```

File: docs/docs/node-creation.md [hunk 4]
```diff
@@ -30,13 +30,13 @@ An important note here is that we do not store a distinct collection of each typ
 
 ### Explicitly recording a parent/child relationship
 
-This occurs when a transformer plugin implements [onCreateNode](/docs/node-apis/#onCreateNode) in order to create some child of the originally created node. In this case, the transformer plugin will call [createParentChildLink](/docs/actions/#createParentChildLink), with the original node, and the newly created node. All this does is push the child's node ID onto the parent's `children` collection and resave the parent to redux.
+This occurs when a transformer plugin implements [onCreateNode](/docs/node-apis/#onCreateNode) in order to create some child of the originally created node. In this case, the transformer plugin will call [createParentChildLink](/docs/actions/#createParentChildLink), with the original node, and the created node. All this does is push the child's node ID onto the parent's `children` collection and resave the parent to redux.
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]:
```suggestion
This occurs when a transformer plugin implements [onCreateNode](/docs/node-apis/#onCreateNode) in order to create some child of the originally created node. In this case, the transformer plugin will call [createParentChildLink](/docs/actions/#createParentChildLink), with the original node, and the newly created node. All this does is push the child's node ID onto the parent's `children` collection and resave the parent to redux.
```

Not sure if we need to remove this?

File: docs/docs/node-creation.md [hunk 5]
```diff
@@ -51,25 +51,25 @@ Let's say you create the following node by passing it to `createNode`
 }
 ```
 
-The value for `baz` is itself an object. That value's parent is the top level object. In this case, Gatsby simply saves the top level node as is to redux. It doesn't attempt to extract `baz` into its own node. It does however track the subobject's root NodeID using [Node Tracking](/docs/node-tracking/)
+The value for `baz` is itself an object. That value's parent is the top level object. In this case, Gatsby simply saves the top level node as is to redux. It doesn't attempt to extract `baz` into its node. But it tracks the subobject's root NodeID using [Node Tracking](/docs/node-tracking/)
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]: I would revert this change? I don't think this is in the scope of the issue that we mean to address?

File: docs/docs/node-creation.md [hunk 6]
```diff
@@ -51,25 +51,25 @@ Let's say you create the following node by passing it to `createNode`
...
 
 During schema compilation, Gatsby will infer the sub object's type while [creating the gqlType](/docs/schema-gql-type#plain-object-or-value-field).
 
 ## Fresh/stale nodes
 
-Every time a build is re-run, there is a chance that a node that exists in the redux store no longer exists in the original data source. E.g. a file might be deleted from disk between runs. We need a way to indicate that fact to Gatsby.
+Every time a build is re-run, there is a chance that a node that exists in the redux store no longer exists in the original data source. E.g. a file might be deleted from disk between runs. We need a way to state that fact to Gatsby.
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]: See comment above

File: docs/docs/node-creation.md [hunk 7]
```diff
@@ -51,25 +51,25 @@ Let's say you create the following node by passing it to `createNode`
...
 
 When a `source-nodes` plugin runs again, it generally recreates nodes (which automatically touches them too). But in some cases, such as [transformer-screenshot](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-screenshot/src/gatsby-node.js#L56), a node might not change, but we still want to keep it around for the build. In these cases, we must explicitly call `touchNode`.
 
-Any nodes that aren't touched by the end of the `source-nodes` phase, are deleted. This is performed via a diff between the `nodesTouched` and `nodes` redux namespaces, in [source-nodes.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/source-nodes.js)
+Any nodes that aren't touched by the end of the `source-nodes` phase are deleted. This is performed via a diff between the `nodesTouched` and `nodes` redux namespaces, in [source-nodes.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/source-nodes.js)
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]:
```suggestion
Any nodes that aren't touched by the end of the `sourceNodes` phase are deleted. This is performed via a diff between the `nodesTouched` and `nodes` redux namespaces, in [source-nodes.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/source-nodes.js)
```

File: docs/docs/node-creation.md [hunk 8]
```diff
@@ -51,25 +51,25 @@ Let's say you create the following node by passing it to `createNode`
...
+From a site developer's point of view, nodes are immutable. In the sense that if you simply change a node object, other parts of Gatsby will not see those changes. To make a change to a node, it must be persisted to redux via an action.
 
-So, how do you add a field to an existing node? E.g. perhaps in onCreateNode, you want to add a transformer specific field? You can call [createNodeField](/docs/actions/#createNodeField) and this will simply add your field to the node's `node.fields` object and then persists it to redux. This can then be referenced by other parts of your plugin at later stages of the build.
+So, how do you add a field to an existing node? E.g. in onCreateNode, you want to add a transformer specific field? You can call [createNodeField](/docs/actions/#createNodeField) and this will simply add your field to the node's `node.fields` object and then persists it to redux. Then the other parts of your plugin at later stages of the build can reference this.
```
Comment (sidharthachatterjee) [inline docs/docs/node-creation.md]: Unsure if this is necessary? I personally find the current state more readable 🤷‍♂

File: docs/docs/node-interface.md [hunk 9]
```diff
@@ -23,6 +23,8 @@ internal: {
 ...other fields specific to this type of node
 ```
 
+## Node Types
```
Comment (sidharthachatterjee) [inline docs/docs/node-interface.md]:
```suggestion
## Node attributes
```

File: docs/docs/node-interface.md [hunk 10]
```diff
@@ -35,27 +37,27 @@ The digest should be unique to the content of this node since it's used for cach
 
 ### `mediaType`
 
-Optional [media type](https://en.wikipedia.org/wiki/Media_type) to indicate to transformer plugins this node has data they can further process.
+Optional [media type](https://en.wikipedia.org/wiki/Media_type) to notify the transformer plugins this node has data they can further process.
```
Comment (sidharthachatterjee) [inline docs/docs/node-interface.md]: Unsure if this is necessary

File: docs/docs/node-interface.md [hunk 11]
```diff
@@ -35,27 +37,27 @@ The digest should be unique to the content of this node since it's used for cach
...
+Stores which plugins created which fields. Gatsby adds this field itself (not the plugin).
 
 ### `content`
 
 Optional field exposing the raw content for this node that transformer plugins can take and further process.
 
 ## Source plugins
 
-New nodes are added to Gatsby by "source" plugins. A common one that many Gatsby
+`Source` plugins adds new nodes to Gatbsy. A common one that many Gatsby
```
Comment (sidharthachatterjee) [inline docs/docs/node-interface.md]:
```suggestion
Source plugins add new nodes to Gatsby. A common one that many Gatsby
```

File: docs/docs/node-tracking.md [hunk 12]
```diff
@@ -47,15 +47,15 @@ Its sub objects are `foo` (value = `{ myfile: "blog/my-blog.md", b: 2}`), and th
 
 ## Find Root Nodes
 
-To access this information, `node-tracking.js` provides the [findRootNodeAncestor()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/schema/node-tracking.js#L52) function. It takes an object, and looks up its parent's nodeID in `rootNodeMap`. It then finds the actual node in redux. It then gets that node's `parent` ID, and gets the parent node from redux. And continues in this way until the root node is found.
+To access this information, `node-tracking.js` provides the [findRootNodeAncestor()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/schema/node-tracking.js#L52) function. It takes an object, and looks up its parent's nodeID in `rootNodeMap`. It then finds the actual node in redux. It then gets that node's `parent` ID, and gets the parent node from redux. And continues in this way until it founds the root node.
```
Comment (sidharthachatterjee) [inline docs/docs/node-tracking.md]:
```suggestion
To access this information, `node-tracking.js` provides the [findRootNodeAncestor()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/schema/node-tracking.js#L52) function. It takes an object, and looks up its parent's nodeID in `rootNodeMap`. It then finds the actual node in redux. It then gets that node's `parent` ID, and gets the parent node from redux. And continues in this way until it finds the root node.
```

File: docs/docs/node-tracking.md [hunk 13]
```diff
@@ -47,15 +47,15 @@ Its sub objects are `foo` (value = `{ myfile: "blog/my-blog.md", b: 2}`), and th
...
+Where is node-tracking used? First up, nodes are tracked in 2 places. Firstly, in [createNode](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/redux/actions.js#L539), every time a node is created, Gatsby links all its subobjects to the new NodeID. Nodes are also tracked whenever they are resolved in [run-sift](/docs/schema-sift/#3-resolve-inner-query-fields-on-all-nodes). This is necessary because [custom plugin fields](/docs/schema-input-gql/#inferring-input-filters-from-plugin-fields/) might return new objects that weren't created when the node was initially made.
 
-Now, where do we use this information? In 2 places.
+Now, where do you use this information? In 2 places.
```
Comment (sidharthachatterjee) [inline docs/docs/node-tracking.md]:
```suggestion
Now, where does Gatsby use this information? In 2 places.
```

File: docs/docs/node-tracking.md [hunk 14]
```diff
@@ -47,15 +47,15 @@ Its sub objects are `foo` (value = `{ myfile: "blog/my-blog.md", b: 2}`), and th
...
 
-1. In the `File` type resolver. It is used to lookup the node's root, which should be of type `File`. We can then use that root node's base directory attribute to create the full path of the resolved field's value, and therefore find the actual `File` node that the string value is describing. See [File GqlType inference](/docs/schema-gql-type/#file-types) for more info.
+1. In the `File` type resolver. It is used to lookup the node's root, which should be of type `File`. You can then use that root node's base directory attribute to create the full path of the resolved field's value, and therefore find the actual `File` node that the string value is describing. See [File GqlType inference](/docs/schema-gql-type/#file-types) for more info.
```
Comment (sidharthachatterjee) [inline docs/docs/node-tracking.md]:
```suggestion
1. In the `File` type resolver. It is used to lookup the node's root, which should be of type `File`. Gatsby can then use that root node's base directory attribute to create the full path of the resolved field's value, and therefore find the actual `File` node that the string value is describing. See [File GqlType inference](/docs/schema-gql-type/#file-types) for more info.
```

File: docs/docs/node-creation.md (hunk 8 above)
Comment (brkn) [reply docs/docs/node-creation.md]: I thought it would be better to convert from **passive voice**.

> Other parts can reference this
Sounded rather cleaner to me than:
> This can be referenced by other parts

I'm not a native speaker, and I'm not sure about any of these changes. Please let me know which one is actually better, I will revert it asap.

File: docs/docs/node-creation.md (hunk 4 above)
Comment (brkn) [reply docs/docs/node-creation.md]: Me niether. Will revert this.

File: docs/docs/node-creation.md (hunk 5 above)
Comment (brkn) [reply docs/docs/node-creation.md]: You are right, reverting this.

File: docs/docs/node-creation.md (hunk 6 above)
Comment (brkn) [reply docs/docs/node-creation.md]: Reverting it.

**Comment by marcysutton** [comment]
Hey! Thanks so much for opening a pull request!

We really appreciate you sending this over, but the changes you’ve proposed are not going to be accepted because they don't meaningfully improve docs. We would recommend opening a new PR with changes to the `node-tracking.md` doc to change the word "we" into "Gatsby", and omit the other changes as they don't follow the Style Guide.

We absolutely want to have you as a contributor, so please take a look at [our open issues][open-issues] for ideas, and please reach out to us [on Twitter at @gatsbyjs](https://twitter.com/gatsbyjs) with questions.
 
Thanks again, and we look forward to seeing more PRs from you in the future! 💪💜
[open-issues]: https://github.com/gatsbyjs/gatsby/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

**Comment by brkn** [comment]
Hi Marcy,

Thanks for feedback, but can you elaborate a bit more about this,
> omit the other changes as they don't follow the Style Guide.

Is this the correct link to [Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)? 

I tried to implement these points from styleguide in this pr:
- [Use active voice](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-active-voice). 
- [Avoid emojis, slang, and metaphors](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#avoid-emojis-slang-and-metaphors)

Will open a new pr very soon, thanks again.
